library/maven

```diff
@@ -17,7 +18,7 @@ Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
 GitCommit: 8f9adaf3d78df6ed956b0a717d51876f71f0fdfe
 Directory: eclipse-temurin-11-focal
 
-Tags: 3.9.1-eclipse-temurin-17, 3.9.1, 3.9.1-eclipse-temurin, 3.9-eclipse-temurin-17, 3.9, 3.9-eclipse-temurin, 3-eclipse-temurin-17, 3, latest, 3-eclipse-temurin, eclipse-temurin
+Tags: 3.9.1-eclipse-temurin-17, 3.9.1, 3.9-eclipse-temurin-17, 3.9, 3-eclipse-temurin-17, 3, latest
```

Comment:
Just to double check, these other generic tags were left on jdk17 on purpose or are they supposed to move as well?

`3.9.1, 3.9, 3, latest`

Reply:
no, that was an oversight, I've fixed my script